configure/executors/executor/executor.Deployment.yaml

```diff
@@ -28,25 +28,21 @@ spec:
     spec:
       containers:
         - name: executor
-          image: index.docker.io/sourcegraph/executor:insiders@sha256:dfeef2e31d6c7b9bc3e5bf581180668f7c033ffcf1fff9d3d6380b7b998d4c2b
+          image: index.docker.io/sourcegraph/executor:insiders@sha256:deda81624796df31b56bdaa6820b3332eaa2cbe5e1729886d15ce3c6f57b69b5
           imagePullPolicy: Always
           livenessProbe:
-            httpGet:
-              path: /healthz
-              port: debug
-              scheme: HTTP
+            exec:
+              command:
+                - /usr/local/bin/executor
+                - validate
             initialDelaySeconds: 60
-            timeoutSeconds: 5
+            timeoutSeconds: 30
           readinessProbe:
-            httpGet:
-              path: /ready
-              port: debug
-              scheme: HTTP
-            periodSeconds: 5
-            timeoutSeconds: 5
-          ports:
-            - containerPort: 6060
-              name: debug
+            exec:
+              command:
+                - /usr/local/bin/executor
+                - validate
```

Comment:
if sourcegraph is down, this would also kill executors _I think_, would that be ok?

Reply:
Bit of a chicken and egg scenario, if SG is down presumably this isn't working anyway. And otherwise there isn't really a nice way to see if this is ready for work. `pgrep <large tini command>` is ugly and we don't open a port. The fact that if the frontend for example is down...we also lose gitserver,repo-updater and a host of other services via readiness checks was my reasoning here. 

Thoughts?

Reply:
Yeah, no strong feelings on this, I don't _love_ that a frontend outage would mean other things cascadingly also go down fully, but nbd either.

Reply:
news flash, found a better pgrep command